pynetdicom/utils.py

```diff
@@ -183,3 +183,13 @@ def validate_uid(uid):
         return True
 
     return False
+
+
+def make_target(target_fn):
```

Comment:
```suggestion
# optionally monkey patches contextvars into the target_fn (requires Python 3.7+).
# See also: _config.py
def make_target(target_fn):
```
thought: suggesting some clarifications as to why this is necessary in the first place. Zero offense taken if this suggestion is rejected or completely reworded.

Reply:
added a docstring, thanks